### PR TITLE
[plugin.video.wnbc] 3.0.18

### PR DESCRIPTION
**Pull request by learningit**
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

**Comment by learningit** [comment]
No idea why there were silly conflicts. I resolved them in the editor.

File: plugin.video.wnbc/addon.xml [hunk 1]
```diff
@@ -20,6 +20,6 @@
     <license>GNU GENERAL PUBLIC LICENSE. Version 2, June 1991</license>
     <forum>http://forum.kodi.tv/showthread.php?tid=234448</forum>
     <website>www.nbc.com</website>
-    <source>https://github.com/learningit/Kodi-plugins-source/tree/master/plugin.video.wnbc</source>
+    <source>https://github.com/learningit/repo-plugins/</source>
```
Comment (Rechi) [inline plugin.video.wnbc/addon.xml]: Is this the repo you want contributors to submit their code changes?

**Comment by Rechi** [comment]
> Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Please squash your commits.

File: plugin.video.wnbc/addon.xml (hunk 1 above)
Comment (learningit) [reply plugin.video.wnbc/addon.xml]: Yes. I only maintain a single fork repo because it's a terror maintaining separate repos for 60 add-ons in GitHub.

**Comment by learningit** [comment]
If you look at the second commit it's to resolve some minor conflict issues due to a bad rebase (GitHub sometimes does strange things with rebase). The resolutions were done in the GitHub editor and as such I can't do a squash.
Both of these issues have been explained in the past.

**Pull request merged**